README.md

```diff
@@ -45,12 +45,10 @@ coming soon
 
 # Steps
 
-The setup is done in 3 primary steps.  You will download the code, setup the application and then deploy the code to IBM Cloud.  If you would like to run the code locally, there will be one more step to configure the credentials locally.
-
 1. [Clone the repo](#1-clone-the-repo)
 2. [Create Watson Studio Project](#2-create-watson-studio-project)
-3. [Create DB2 warehouse on cloud and add the connection to Watson Studio](#3-create-db2-warehouse-on-cloud-and-add-the-connection-to-watson-studio)
-4. [Create and load data into DB2 warehouse](#4-create-and-load-data-to-db2-warehouse)
+3. [Create Db2 Warehouse on Cloud database and add the connection to Watson Studio](#3-create-db2-warehouse-on-cloud-database-and-add-the-connection-to-watson-studio)
```

Comment:
Notice here you went from cloud to Cloud, but above you went from Cloud to cloud.  I think it is cap but whatever it is do a search and make it consistent.